docs/en/latest/plugins/key-auth.md

```diff
@@ -41,6 +41,7 @@ For route side:
 | ---- | ------ | ----------- | ------- | ----- | ---------------------------------------------------------------------------- |
 | header  | string | optional    | apikey        |       | the header we get the key from |
 | query   | string | optional    | apikey        |       | the query string we get the key from, which priority is lower than `header` |
+| hide_credentials   | bool | optional    | false        |       | Whether to pass the apikey request headers to the upstream. |
```

Comment:
It's confusing, the header is not necessary to be "apikey", it's specified by the `header`. Also, the query should also be removed.

Reply:
ok

Reply:
Query is the content modified by others. Can I delete it?

Reply:
You should only handle the query that used to carry the credential.